int32 fiducial_id
---
